pages/create-expense.js

```diff
@@ -19,6 +19,7 @@ import CollectiveNavbar from '../components/collective-navbar';
 import { collectiveNavbarFieldsFragment } from '../components/collective-page/graphql/fragments';
 import Container from '../components/Container';
 import ContainerOverlay from '../components/ContainerOverlay';
+import { SignInOverlayBackground } from '../components/conversations/CommentForm';
```

Comment:
It would be more logical IMO to have `SignInOverlayBackground` in `components/SignInOrJoinFree.js` or eventually in its own file (i.e. `components/SignInOverlayBackground.js`).

Reply:
I've moved this to the `SignInOrJoinFree.js` page. Thanks. 👍🏽